lib/plugins/wallet/geth/base.js

```diff
@@ -115,29 +117,37 @@ function generateTx (_toAddress, wallet, amount, includesFee, cryptoCode) {
 
   if (isErc20Token) txTemplate.data = contractData
 
+  const common = new Common({ chain: _.defaultTo(Chain.Ropsten, opts?.chainId), hardfork: Hardfork.London })
```

Comment:
```suggestion
  const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London })
```

Why do we use _.defaultTo here? I don't think the Common constructor will accept anything other than strings or enums on the chain field.

Reply:
It's an enum that corresponds to the chain ID. So far, it's working great, should it be removed?